Source/WebKit/WebProcess/GPU/graphics/RemoteGraphicsContextGLProxy.cpp

```diff
@@ -289,6 +289,63 @@ void RemoteGraphicsContextGLProxy::simulateEventForTesting(SimulatedEventForTest
     }
 }
 
+void RemoteGraphicsContextGLProxy::getBufferSubData(GCGLenum target, GCGLintptr offset, std::span<uint8_t> data)
+{
+    if (isContextLost())
+        return;
+
+    if (data.empty())
+        return;
+
+    static constexpr size_t getBufferSubDataInlineSizeLimit = 64 * KB; // NOTE: when changing, change the value in RemoteGraphicsContextGL too.
+    static constexpr size_t getBufferSubDataSharedMemorySizeLimit = 100 * MB;
+
+    auto copyToData = [&](ptrdiff_t offset, std::span<const uint8_t> replyData) {
+        memcpy(data.data() + offset, replyData.data(), replyData.size_bytes());
+    };
+
+    ptrdiff_t transferOffset = 0;
+    size_t transferRemaining = data.size();
+    size_t transferSize = 0;
```

Comment:
transferSize can and should be  a variable local to the scope it's used in.

Reply:
It's used in the inline case if grabbing a handle suddenly fails to transfer the remaining data via 64 KB chunks inline.